Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -150,26 +164,24 @@ public final class DiskMemoryPersistenceStack: NSObject, PersistenceStack {
 
     private func cachedData(for key: Persistence.Key) -> Data? {
 
-        let identifier = "\(Constants.PerformanceMetrics.readMemoryKey) \(key)"
+        let identifier = persistencePerformanceMetrics?.readMemoryEventKey ?? ""
 
-        performanceMetrics?.begin(with: identifier)
+        persistencePerformanceMetrics?.metrics.begin(with: identifier)
         let data = cache.object(forKey: key.nsString) as Data?
-        performanceMetrics?.end(with: identifier)
+        persistencePerformanceMetrics?.metrics.end(with: identifier)
```

Comment:
shouldn't/couldn't we also associate the blob's size?

Reply:
Fixed